### ISSUE TITLE
Add "max" data series to the resource monitoring dashboard

### ISSUE THREAD

**Issue by sidoruka** [epam/cloud-pipeline#1276]
**Background**
At the moment, Cloud Pipeline displays the resources utilization as an `average`. This can hide some spikes (which resulted in job failure), when reviewing at a high zoom-level (e.g. for 5 days).

**Approach**
* Let's introduce another data-series ("line"), which is calculated as a `max` function
* Existing one shall be kept as well, but renamed to contain `average` keyword

**Comment by rodichenko** [comment]
To display `max` values we need to receive this data from API (e.g., we need to receive `{cpuUsage: {load, max}, memoryUsage: {capacity, usage, max}}` instead of `{cpuUsage: {load}, memoryUsage: {capacity, usage}}`. @mzueva could you please handle this?

**Comment by mzueva** [comment]
@rodichenko `max` was added to `cpuUsage` and `memoryUsage`

**Comment by rodichenko** [comment]
@sidoruka GUI part implemented (#1305)

**Comment by sidoruka** [comment]
@NShaforostov waiting for your review

**Comment by NShaforostov** [comment]
@rodichenko I've tried to verify. In general, it works well.
A little question:
I see the difference between "real" and "average" lines on a large scale, but these both lines are being merge together once a scale is small (when the delimiter becomes 1 minute or smaller), e.g.:
- ![image](https://user-images.githubusercontent.com/45459424/91091401-32654280-e65f-11ea-8f56-b7595982aaf3.png)
(delimiter is 2 minutes, lines are separate)

- ![image](https://user-images.githubusercontent.com/45459424/91091485-4f017a80-e65f-11ea-90a6-14a89666cb2f.png)
(delimiter is 1 minute, lines are being merged)

Is it normal behavior?

**Comment by rodichenko** [comment]
@NShaforostov
**TL;DR** Yes, this is absolutely normal, because
```
average(a) = maximum(a)
average(a, b, c, ....) <= maximum(a, b, c, ...)
```
**...in other words...**
`Average` data series consists of data points of aggregated data, where each point is an *average* value for specified tick duration - 1 minute, 2 minutes, hour etc; the `real` (or `max`) data series have data points, where each point is the maximum of that tick duration.
Original data is *discrete* meaning that data is collected every **N** seconds or minutes, let's say - 1 minute (I'm not sure what exact time range it is); that means that we cannot get resources utilization for every 1 second - there is no data for that. Considering that we have original data points of every 1 minute:
* if you are requesting data for 30 minutes tick range / delimiter, every chart data point will have an aggregation for 30 data points, and
```
average(30 points) <= maximum(30 points)
```
and `average` line will be "under" the `max` line;
* if you are requesting data for 1 minutes tick range / delimiter, every chart data point will have an aggregation for 1 data point, and
```
average(1 point) = maximum(1 point)
```
and `average` line will be the same as the `max` one.

**Comment by NShaforostov** [comment]
@rodichenko 
Thanks for the detailed answer. I supposed something like that but thought that minimal tick duration is less than a minute.
In such case, this issue is fully verified, all works nice.

**Comment by sidoruka** [comment]
@rodichenko @mzueva please backport to `release/0.16`

**Comment by rodichenko** [comment]
@sidoruka GUI part backported (a059d79729fe6f65e2503deb1dbd8c1a92121f9e)

**Comment by mzueva** [comment]
@sidoruka backported changed to release/0.16 via  786b65d99317fcbc408b4ea12ff62c1fe57b908d fdebac426bf73f0cb173bcb5915b0be37dc88be7

**Comment by NShaforostov** [comment]
Docs were added by #1423
